### ISSUE TITLE
Div by zero

### ISSUE THREAD

**Issue by Vasiliy566** [RaduG/pytest_func_cov#10]
Div by zero error if no functions was detected
```
    raise ex[1].with_traceback(ex[2])
  File "/home/vasily/work/leader-invest-backend/venv/lib/python3.8/site-packages/pluggy/callers.py", line 187, in _multicall
    res = hook_impl.function(*args)
  File "/home/vasily/work/leader-invest-backend/venv/lib/python3.8/site-packages/pytest_func_cov/plugin.py", line 148, in pytest_terminal_summary
    total_cover = int(((total_funcs - total_miss) / total_funcs) * 100)
ZeroDivisionError: division by zero
```

**Comment by Vasiliy566** [comment]
fixed in https://github.com/RaduG/pytest_func_cov/pull/9

**Comment by RaduG** [comment]
Good spot! Thanks

**Comment by RaduG** [comment]
Fixed by #9
